gradle-baseline-java-config/resources/checkstyle/checkstyle.xml

```diff
@@ -127,6 +127,10 @@
             <property name="illegalPkgs" value="junit.framework"/>
             <message key="import.illegal" value="Use JUnit 4-style (org.junit.*) test classes and assertions instead of JUnit 3 (junit.framework.*)."/>
         </module>
+        <module name="IllegalImport"> <!-- Some OpenJDK builds do not include javafx -->
```

Comment:
Can you also give this an ID e.g. `<property name="id" value="BanJavaFx"/>`

Reply:
done